### PR TITLE
BETA: Register fathulfahmy.is-a.dev

### PR DESCRIPTION
**Pull request by fathulfahmy**
Added `fathulfahmy.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/fathulfahmy.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "fathulfahmy",
+        "email": "mfathulfahmy@gmail.com"
+    },
+    "record": {
+        "URL": "fathulfahmy.netlify.app"
```
Comment (Stef-00012) [inline domains/fathulfahmy.json:7]: Invalid URL record (missing protocol)

**Review by Stef-DP — CHANGES_REQUESTED** [review]

File: domains/fathulfahmy.json (hunk 1 above)
Comment (fathulfahmy) [reply domains/fathulfahmy.json]: Thank you for notifying me, I have updated the url with `https://` protocol

**Comment by Stef-00012** [comment]
seems good now

**Review by iostpa — APPROVED** [review]
seems good to me

**Pull request merged**